### ISSUE TITLE
Adding image with Caption to a Page with Pods Shortcode strips HTML from output template

### ISSUE THREAD

**Issue by PodsBot** [pods-framework/pods#3814]
Adding image with Caption to a Page with Pods Shortcode strips HTML from output template submitted via Slack by jimtrue

**Comment by chadwilliams** [comment]
I had a working Pods configuration (JSON export provided) that displayed my template info on a Page. I then added a standard image through the WordPress Add Media functionality and supplied Title, Caption and Alt info, but then all the Pods Template HTML formatting was stripped out leaving just raw text. Unfortunately I had also just updated a few plugins and added a new one so adding this image didn't jump out to me as the problem, but that was exactly the case. After deactivating all plugins and resorting back to the default Twentysixteen theme I still had the issue. I deleted the new plugin I installed and this wasn't the issue. I also added shortcodes=1 as seen here: [pods name="salad_soup" orderby="ID" limit="-1" template="Salad Soup Template" shortcodes=1] as well as define( 'PODS_SHORTCODE_ALLOW_SUB_SHORTCODES', true ); and the problem still existed. Finally I dropped my database and imported a backup copy and it was working fine yet again. However, when I added the image back with the caption it stripped the HTML yet again.

Next I imported the database copy again, but this time when I added the image I didn't include a Caption and it worked.

Finally, I created a new Page, added an image with a Caption and the Pods Shortcode from above and it stripped the HTML yet again, however, the other page that I had just fixed and only added the image without the Caption was still displaying my Pods Template as expected.

So this is a Page by Page issue when a Caption is used like so:

[caption id="attachment_203" align="alignnone" width="760"]<img src="/wp-content/uploads/IMG_0292-1.jpg" alt="Nicks" width="760" height="507" class="size-full wp-image-203" /> Nicks[/caption]

Note the code above was generated directly from the WordPress Media Gallery when adding an image.

Also note that on that page if you just remove the Caption shortcode info from around the image and updating your page the formatting still won't work.

[json-export.txt](https://github.com/pods-framework/pods/files/520150/json-export.txt)
